selfdrive/car/toyota/values.py

```diff
@@ -55,6 +55,7 @@ class CAR:
   LEXUS_ESH_TSS2 = "LEXUS ES HYBRID 2019"
   LEXUS_IS = "LEXUS IS 2018"
   LEXUS_NX = "LEXUS NX 2018"
+  LEXUS_NX = "LEXUS NX 2019"
```

Comment:
This isn't needed and should be removed. The year doesn't matter as long as the platform is the same

Reply:
```suggestion
```